runtime/moonbeam/tests/integration_test.rs

```diff
@@ -34,6 +38,58 @@ use precompiles::MoonbeamPrecompiles;
 use sp_core::{Public, H160, U256};
 use sp_runtime::DispatchError;
 
+#[test]
+fn verify_pallet_index() {
+	assert_eq!(System::index(), 0);
+	assert_eq!(moonbeam_runtime::Utility::index(), 1);
```

Comment:
IMO we should hard code indices in `consrtuct_runtime!` for Moonbeam like we did for Moonriver.

Reply:
Agreed, I expect we will